app/src/main/java/space/taran/arknavigator/ui/fragments/FoldersFragment.kt

```diff
@@ -63,7 +63,7 @@ class FoldersFragment: MvpAppCompatFragment(), FoldersView, BackButtonListener {
 
     override fun init() {
         Log.d(FOLDERS_SCREEN, "initializing FoldersFragment")
-        (activity as MainActivity).setSelectedTab(0)
+        (activity as MainActivity).setSelectedTab(1)
```

Comment:
Do these numbers depend on order of initialization somewhere else?
If we can make them explicit that would be great.
If no, just define constants somewhere near to the place where they are determined.